CHANGELOG

```diff
@@ -1,3 +1,127 @@
+== OpenDataPlane (1.19.0.1)
+=== Summary of Changes
+ODP v1.19.0.1 is the first service update for the Tiger Moth release. It
+incorporates a number of corrections and enhancements that improve the quality
+and testability of ODP.
+
+==== APIs
+There are no API changes in this release.
+
+==== DPDK Service Release Sync
+ODP now uses DPDK 17.11.2 for it's DPDK-related support. This is the current
```

Comment:
is now tested with/works with. ODP per se does not use DPDK

Reply:
`odp-linux` uses DPDK for DPDK PktIOs. `odp-dpdk` uses it throughout. This changelog will apply to both reference implementations.

Reply:
I'd still suggest the following wording: `ODP is not tested against DPDK 17.11.2...`